ember-cli-build.js

```diff
@@ -85,10 +85,12 @@ module.exports = function(_options) {
   /*
    * ES6 Build
    */
-  var tokenizerPath = path.join(require.resolve('simple-html-tokenizer'), '..', '..', 'src');
-  // TODO: WAT, why does { } change the output so much....
-  var tokenizerTree = find(tokenizerPath, { });
-  var HTMLTokenizer = mv(tokenizerTree, "simple-html-tokenizer");
+  var tokenizerPath = path.join(require.resolve('simple-html-tokenizer'), '..', 'es6');
+  var HTMLTokenizer = find(tokenizerPath, {
+    include: ['**/*.js'],
+    exclude: ['**/*.d.ts']
+  });
+  HTMLTokenizer = mv(htmlTokenizerTree, 'simple-html-tokenizer');
```

Comment:
@locks why didn't you keep the old names and where is `htmlTokenizerTree` coming from?